spring-integration-mail/src/main/java/org/springframework/integration/mail/dsl/ImapMailInboundChannelAdapterSpec.java

```diff
@@ -65,4 +65,19 @@ public ImapMailInboundChannelAdapterSpec shouldMarkMessagesAsRead(boolean should
 		return this;
 	}
 
+	/**
+	 * How often to recycle the idle task (in case of a silently dropped connection).
+	 * Seconds; default 120 (2 minutes).
+	 * @param interval the interval.
+	 * @return the spec.
+	 * @see ImapMailReceiver#setCancelIdleInterval(long)
+	 * @since 5.0.10
+	 */
+	public ImapMailInboundChannelAdapterSpec cancelIdleInterval(long interval) {
+		assertReceiver();
+		this.receiver.setCancelIdleInterval(interval);
```

Comment:
How do you think: don't we have anything else missed to be exposed?

Reply:
These are the only 3 properties on the `ImapMailReceiver` (there are only 3 setters).